liveblog/liveblog
# [LBSD-2808] - Updating themes doesn't update embed

### Purpose

This PR fixes a bug which was causing the public URL to remain unchanged even after the theme was updated.

### What has changed

- Changed the `publish_blog_embeds_on_s3` save argument to True.
- Added test for `publish_blog_embeds_on_s3`, `publish_blog_embed_on_s3` and `internal_publish_blog_embed_on_s3`
- Ensured that the main public URL does not include the theme. The theme is only included in the output channel links.

### Steps to test

1. Log in to the Editor. Click Live to open the preview and observe the current theme.
2. Go to the Blog Settings and under the Theme option, select a different theme.
3. Click Save to save the settings. 
4. Go back the Editor and click Live. Observe that the theme has changed. 


Resolves: [LBSD-2808](https://sofab.atlassian.net/browse/LBSD-2808)

[LBSD-2808]: https://sofab.atlassian.net/browse/LBSD-2808?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
